geoapi/services/projects.py

```diff
@@ -217,8 +224,11 @@ def getFeatures(projectId: int, query: dict = None) -> object:
         if startDate and endDate:
             sub_select = sub_select.where(text("feat.created_date BETWEEN :startDate AND :endDate"))
 
-        if assetType:
-            sub_select = sub_select.where(text("fa.asset_type = :assetType"))
+        if len(assetQueries):
+            sub_select = sub_select.where(text('(' + ' OR '.join(assetQueries) + ')'))
+        else:
+            params['none'] = 'none'
+            sub_select = sub_select.where(text('fa.asset_type = :' + params['none']))
```

Comment:
```suggestion
```
also, for the case where there is no assetType filter (i.e. /api/projects/2/features/) and we want a complete listing of features regardless of asset types